biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -74,11 +77,11 @@ export class BioBattery implements Battery {
     return validate(batteryValidator);
   }
 
-  getCurrentBatteryEnergy(): Energy {
+  getEnergyInJoules(): Energy {
     return this.currentBatteryEnergy;
   }
 
-  getMaxcapacity(): Energy {
+  get MaxCapacity(): Energy {
```

Comment:
why are we renaming this? getters should be named getVariableName

Reply:
This is used in the biobrain algorithm in method determineSupplyingPath to find the energy required by the receiving agent.
I changed the getter name so that I don't have to check which gridItem it is since all are expected to have this getter method MaxCapacity. Also getter methods are different in typescript, you can write `get` then following by the name

Reply:
I understand that this does work, but we want to be consistent in our code. The rest of this code base doesn't use the get <name> method, so we should continue that pattern throughout

Reply:
I agree with Vita on the getters. Otherwise, all looks good pending changes